src/buza/managers.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from django.contrib.auth.base_user import BaseUserManager
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+
+
+def validate_phone(phone):
+    if phone.startswith("+27"):
+        phone_pattern = re.compile(r'[+]27(\d{9})$')
+        return phone_pattern.match(phone)
+    else:
+        phone_pattern = re.compile(r'0(\d{9})$')
+        return phone_pattern.match(phone)
```

Comment:
I would avoid regex-based phone number validation, even at first.

We can either leave this unvalidated, for now, or if we do validation, we should do it using something based on Google's libphonenumber, or similar.